.github/workflows/embedded.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           EMBED_USE_LLVM: ON
           RUN_ALL_TESTS: 1
           RUNTIME_TEST_DISABLE: builtin.cgroup,probe.kprobe_offset_fail_size
-          BASE: bionic
```

Comment:
can we keep this on bionic for now? We're dynamic linking against libc so that will probably means that the binary from docker won't work on older systems anymore.

Also can't do this because the LLVM build is broken

Reply:
Sounds good